.ci/packaging.groovy

```diff
@@ -86,6 +86,22 @@ pipeline {
                   'x-pack/winlogbeat'
                 )
               }
+              axis {
+                name 'PLATFORMS'
+                values (
+                  '+all',
```

Comment:
the axis should be like "+all linux/amd64" otherwise the build will not be correct